### DISCUSSION TITLE
Translation of the selection

### DISCUSSION THREAD

**Discussion by Henry2o1o** [farag2/Sophia-Script-for-Windows#243] [Ideas]
![image](https://user-images.githubusercontent.com/6131799/131037022-d43043a8-ef21-4318-a74a-5df43afbc35e.png) Warnung, Überprüfen

👆 
![image](https://user-images.githubusercontent.com/6131799/131037662-94c6f75d-6307-46d2-9eb2-2f9288148377.png)

![image](https://user-images.githubusercontent.com/6131799/131037043-0912c117-c33a-4196-bda3-7a820511c9fc.png) Deaktivieren, Aktivieren, Nichts

Are translations possible in the selection?

**Comment by farag2** [comment]
I'll think it out)

**Comment by farag2** [comment]
Good question

**Comment by farag2** [comment]
Done. :) Will be added in 2.4.
